with-post-step/README.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Generic JS Action to execute a main command and set a command as a post step.
```

Comment:
Copy the important links here too for context:
e.g. https://github.com/actions/runner/issues/1478 and https://github.com/pyTooling/Actions/tree/main/with-post-step